geometry_msgs/PoseStamped pose
sensor_msgs/Image image
